src/lua/zencode_sd_jwt.lua

```diff
@@ -478,3 +478,19 @@ IfWhen("verify signed selective disclosure '' issued by '' is valid", function(o
     end
 
 end)
+
+When("create disclosed informations from signed selective disclosure ''", function(ssd_name)
```

Comment:
s/informations/information/ since there is no plural for that word in english.

Reply:
Outdated